searx/engines/bing_videos.py

```diff
@@ -67,6 +67,10 @@ def request(query, params):
     if params['time_range'] in time_range_dict:
         params['url'] += time_range_string.format(interval=time_range_dict[params['time_range']])
 
+    # bing videos did not like "older" versions < 70.0.1 when selectin other
+    # languages then 'en' .. very strange ?!?!
+    params['headers']['User-Agent'] = 'Mozilla/5.0 (X11; Linux x86_64; rv:73.0.1) Gecko/20100101 Firefox/73.0.1'
```

Comment:
It works, but if Bing keeps the same behaviour (don't kike "older" FF versions), it will not work sooner or later.

For information, see comments about the initial implemantation of PR https://github.com/asciimoo/searx/pull/933 (automatic updates of Firefox version).